anta/models.py

```diff
@@ -34,6 +34,14 @@
 
 logger = logging.getLogger(__name__)
 
+KNOWN_EOS_ERRORS = [
+    r"BGP inactive",
+    r"VRF '.*' is not active",
+    r".* does not support IP",
```

Comment:
This was for VerifyReachability right? If so, I am still getting an error:
`ERROR    [anta.device] Command 'ping vrf MGMT 1.1.1.1 source Management1 size 100 df-bit repeat 2' failed on DC1-LEAF1A: No source interface Management1 `

This happens with an interface not configured on the switch. Another example:
`ERROR    [anta.device] Command 'ping vrf MGMT 1.1.1.1 source Ethernet100 size 100 df-bit repeat 2' failed on DC1-LEAF1A: No source interface Ethernet100 `

Reply:
> r VerifyReachability  error was `r"VRF '.*' is not active",` if you look at theissue

yours is new :p we need to add it but it was never reported